README.rst

```diff
@@ -97,6 +97,25 @@ patched and that you don't have to worry about the details.
 API
 ===
 
+``replace(func, new_source)``
```

Comment:
the reason i put it there is so i could copy/paste from the surrounding paragraphs (rather than having to scroll up and down for every snipped i'd need), since i'm not at all familiar with "pydoc" or whatever that is :)